src/Microsoft.Data.Sqlite.Core/Utilities/DateTimeConversion.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Data.Sqlite
+{
+    using System;
+
+    internal static class DateTimeConversion
+    {
+        // Computes DateTime from julian date. This function is a port of the
+        // computeYMD and computeHMS functions from the original Sqlite core
+        // source code in 'date.c'.
+        public static DateTime FromJulianDate(double julianDate)
+        {
+            // computeYMD
+            long iJD = (long)(julianDate * 86400000.0 + 0.5);
```

Comment:
Our [engineering guidelines](https://github.com/aspnet/Home/wiki/Engineering-guidelines#usage-of-the-var-keyword) encourage the use of `var`.